frontend/src/scenes/ingestion/ingestionLogic.ts

```diff
@@ -106,7 +107,7 @@ export const ingestionLogic = kea<ingestionLogicType>({
             () => [],
             (): boolean => {
                 const featFlags = featureFlagLogic.values.featureFlags
-                return !!featFlags[FEATURE_FLAGS.ONBOARDING_1]
+                return featFlags[FEATURE_FLAGS.ONBOARDING_1] === 'test'
```

Comment:
We might have to update the name of this FF when we set up the experiment. Right now, I don't think you can convert a FF -> Experiment

Reply:
Oh wait, we haven't made this FF on cloud yet, have we?

Reply:
we have not :)

Reply:
good catch though!! 🧐